jsonnet/rules.libsonnet

```diff
@@ -24,7 +24,7 @@ function(params) {
         },
         {
           expr: |||
-            count by(node,workload,namespace)
+            count by(workload,namespace)
```

Comment:
I will update the runbook according to this change in the aggregation since the `node` label will not be exposed anymore in the alert.